bears/python/PEP8Bear.py

```diff
@@ -30,7 +30,6 @@ def run(self, filename, file,
         """
         Detects and fixes PEP8 incompliant code. This bear will not change
         functionality of the code in any way.
-
```

Comment:
Here you should enter a new line after your omission to follow proper indentation.New line should come before ```:param max_line_length:   Maximum number of characters for a line.When set to 0 allows infinite line length. ``` to follow proper indentation.

Reply:
okay